src/carma-platform/guidance_scripts.py

```diff
@@ -1678,4 +2024,15 @@ def run_turn_speed_analysis(
 
     return (is_passed, speed_excess_stats, plt.gcf(), speed_excess, steer_times)
 
-# More guidance specific analysis scripts to come ....
\ No newline at end of file
+# More guidance specific analysis scripts to come ....
+
+def main():
+    """
+    Main function to run the analysis scripts.
+    """
+    # Example usage of the functions
+    mcap_path = "/workspaces/carma_ws/src/analysis-data/rosbag2_2025-04-11_032029/rosbag2_2025-04-11_032029_0.mcap"
```

Comment:
Nit: Can we add a TODO for the user to replace `mcap_path`?

Reply:
yup, I changed the path